client/lib/abtest/active-tests.js

```diff
@@ -1,5 +1,15 @@
 /** @format */
 export default {
+	springSale30PercentOff: {
+		datestamp: '20180413',
+		variations: {
+			upsell: 50,
+			control: 50,
+		},
+		defaultVariation: 'upsell',
```

Comment:
Should the default be the control instead?
https://github.com/Automattic/wp-calypso/tree/master/client/lib/abtest#dealing-with-ineligible-users